Added the dotnet example

Madhan-Reddy-ni opened the pull request:
**Justification**
In order to represent how to use the APIs created in the TestMonitor service in the SystemLink Enterprise version, there is a need to create examples that will show how to call the APIs, what details are needed to be sent with the API call and etc.

**Implementation**
Since there is already an example available that will use the SLS, I have copied that example and modified it to use the SLE server

**Testing**
Manual testing completed

adamarnesen commented:
There are two image files that are showing screenshots from SLS. We should either remove these or update them to have the new SLE images.

adamarnesen commented:
After talking with @TJ-G today, I think that we want to make a few modifications here to keep our repo structure clean. We want to have one top-level folder for each example. That folder should have a short, descriptive name and a readme.md file that describes the example. The `Simple ETL Example` is the correct pattern to follow here. 

When we have a .sln file and project files like in this one for .NET, we can have the solution file in the top-level of the folder and the project files appropriately related under that.

Madhan-Reddy-ni commented on examples/Test Monitor/NISystemLinkEnterpriseExamples/Results/ExampleConfiguration.cs:
```diff
@@ -0,0 +1,153 @@
+﻿/*
+ * This file supports other examples. See configuration/Configuration.cs for
```
Removed the comment

adamarnesen reviewed (changes requested):
@Madhan-Reddy-ni We don't quite have the structure right here. We don't want to have an `examples` folder. We just want to have a list of folders at the top-level that each have an example in them. The fix is to rename the `example` folder here to something like `TestMonitor dotnet Example`

Madhan-Reddy-ni commented:
@adamarnesen, I have renamed the `examples` folder name to the `DotNet Examples`, since we will be adding more DotNet examples related to other services also(not only Test Monitor) and we can add them inside this folder. We will also be adding the Python examples and we can create one more folder as `Python Examples` and add the python related examples in here.

Madhan-Reddy-ni commented:
> @adamarnesen, I have renamed the `examples` folder name to the `DotNet Examples`, since we will be adding more DotNet examples related to other services also(not only Test Monitor) and we can add them inside this folder. We will also be adding the Python examples and we can create one more folder as `Python Examples` and add the python related examples in here

@adamarnesen, I have renamed the examples folder name to the DotNet Examples, since we will be adding more DotNet examples related to other services also(not only Test Monitor) and we can add them inside this folder. We will also be adding the Python examples and we can create one more folder as Python Examples and add the python related examples in here

Madhan-Reddy-ni commented on examples/Test Monitor/NISystemLinkEnterpriseExamples/Results/Results.cs:
```diff
@@ -0,0 +1,196 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
```
> @Madhan-Reddy-ni We don't quite have the structure right here. We don't want to have an `examples` folder. We just want to have a list of folders at the top-level that each have an example in them. The fix is to rename the `example` folder here to something like `TestMonitor dotnet Example`

@adamarnesen, I have renamed the examples folder name to the DotNet Examples, since we will be adding more DotNet examples related to other services also(not only Test Monitor) and we can add them inside this folder. We will also be adding the Python examples and we can create one more folder as Python Examples and add the python related examples in here.

Madhan-Reddy-ni replied:
Since the package is available online on GitHub, the user can directly restore the project to get the `TestMonitor` client and run the example.

Madhan-Reddy-ni commented:
> @Madhan-Reddy-ni We don't quite have the structure right here. We don't want to have an `examples` folder. We just want to have a list of folders at the top-level that each have an example in them. The fix is to rename the `example` folder here to something like `TestMonitor dotnet Example`

@adamarnesen, I have renamed the examples folder name to the DotNet Examples, since we will be adding more DotNet examples related to other services also(not only Test Monitor) and we can add them inside this folder. We will also be adding the Python examples and we can create one more folder as Python Examples and add the python related examples in here

Madhan-Reddy-ni commented on examples/Test Monitor/NISystemLinkEnterpriseExamples/Results/ExampleConfiguration.cs:
```diff
@@ -0,0 +1,153 @@
+        /// <returns>A configuration to use for the example.</returns>
+        public static IHttpConfiguration Obtain(
+            string[] args,
+            bool allowCloud = true)
```
Re-worked the example configuration to support only SLE

adamarnesen commented on DotNet Examples/README.md:
```diff
@@ -0,0 +1,41 @@
+Unless otherwise specified by the example's README, each example is run in the
+same way:
+
+1. Download and extract the [repository source](https://github.com/ni/systemlink-client-docs/archive/master.zip).
```
```suggestion
1. Clone this repo _or_ download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
```

adamarnesen commented on DotNet Examples/README.md:
```diff
@@ -0,0 +1,41 @@
+Solution File
+-------------
+
+For convenience, the [NISystemLinkClientExamples.sln](NISystemLinkClientExamples.sln)
```
```suggestion
For convenience, the [NISystemLinkEnterpriseExamples.sln](NISystemLinkEnterpriseExamples.sln)
```

adamarnesen commented on DotNet Examples/Test Monitor/README.md:
```diff
@@ -0,0 +1,28 @@
+Unless otherwise specified by the example's README, each example is run in the
+same way:
+
+1. Download and extract the [repository source](https://github.com/ni/systemlink-client-docs/archive/master.zip).
```
```suggestion
1. Clone _or_ download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
```

adamarnesen commented on DotNet Examples/Test Monitor/Results/README.md:
```diff
@@ -0,0 +1,36 @@
+Running the Example
+-------------------
+
+1. Download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
```
```suggestion
1. Clone _or_ download and extract the [repository source](https://github.com/ni/systemlink-enterprise-examples/archive/master.zip).
```

adamarnesen reviewed (approved):
I just had a few minor additional changes to some of the readme files before we submit this.
